packages/storage/src/api.ts

```diff
@@ -76,6 +77,28 @@ export {
 } from './implementation/taskenums';
 export { StringFormat };
 
+/**
+ * Downloads the data at the object's location. Returns an error if the object
+ * is not found.
+ *
+ * To use this functionality, you have to whitelist your app's origin in your
+ * Cloud Storage bucket. See also
+ * https://cloud.google.com/storage/docs/configuring-cors
+ *
+ * @public
```

Comment:
@egilmorez can you please scan the PR and review the comments with the `@public` tag? Thanks!

Reply:
I was hoping @markarndt could give that a go . . . but if he's too busy and there's urgency, I can take a look this afternoon for sure.